### PR TITLE
add update (temp)

### PR DESCRIPTION
**Pull request by lamuria**

### CODE REVIEW

**Comment by donbonifacio** [comment]
What about a test?

**Comment by lamuria** [comment]
LOL have you seen the temp label on PR name? I'm on it :)

**Comment by donbonifacio** [comment]
:+1:

**Comment by lamuria** [comment]
@donbonifacio i'll close this one

File: test/clanhr/postgres_gateway/core_test.clj [hunk 1]
```diff
@@ -168,6 +168,26 @@
         (is (result/succeeded? total))
         (is (= 0 (:data total)))))))
 
+(deftest update-data
+  (let [users (take 10 (repeatedly create-user))
+        users-saved (mapv #(<!! (core/save-model! % {:table table
+                                                     :service-name "delete-data"
```
Comment (donbonifacio) [inline test/clanhr/postgres_gateway/core_test.clj]: delete-data?

**Comment by donbonifacio** [comment]
:ok:

**Pull request merged**